(2) Vox Populi/Balance Changes/Text/en_US/UnitText.xml

```diff
@@ -636,6 +636,13 @@
 		<Row Tag="TXT_KEY_PROMOTION_BARBARIAN_PENALTY_III_HELP">
 			<Text>-10% [ICON_STRENGTH] Combat Strength VS [COLOR_NEGATIVE_TEXT]Barbarians[ENDCOLOR].</Text>
 		</Row>
+		
+		<Row Tag="TXT_KEY_PROMOTION_FROGMAN">
+			<Text>Frogman</Text>
+		</Row>
+		<Row Tag="TXT_KEY_PROMOTION_FROGMAN_HELP">
+			<Text>[COLOR_POSITIVE_TEXT]Embarking[ENDCOLOR] and [COLOR_POSITIVE_TEXT]Disembarking[ENDCOLOR] Costs 1 [ICON_MOVES] Movement.[NEWLINE]Eliminates combat penalties for attacking from the sea or over a river.</Text>
```

Comment:
This promotion also causes crossing a rive to only cost 1 movement.

Reply:
Yes. It’s a pickable promotion for recon line, who get to move everywhere for 1 move cost innately, so I didn’t deem it noteworthy to say that.